src/pages/Product/Popup.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import './Popup.scss';
+
+class Popup extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isPopupOn: false,
+    };
+  }
+
+  togglePopup = () => {
+    this.setState({
+      isPopupOn: !this.state.isPopupOn,
+    });
+  };
+
+  render() {
+    const { isPopupOn } = this.state;
+    const { image, name, price } = this.props;
+
+    return (
+      <div>
```

Comment:
```suggestion
      <>
```
fragment로 쓰시거나
```suggestion
      <div className="modal">
```
className 부여하시고 nesting 하시면 좋을거 같아요